CHANGELOG.md

```diff
@@ -1,6 +1,4 @@
-## v3.0.0:
```

Comment:
Shouldn't there still be a changelog entry for 3.0.0?

Reply:
I "forgot" for the release, but honesty, the changes were so enormous that it would be a) too much work and b) it should just be considered a new software package essentially. MIght fix it once we do a `v3.0.1` to at least state this instead of having the todo, but I wasn't planning on actually trying to write the changelog

Reply:
Ha, fair enough 👍 Maybe you can just condense the "backwards-incompatible changes" issue?

Reply:
```
## v3.0.0:

The fire nation attacked.
```

Reply:
Yeah, I thought about including the items from that issues, but the problem is that it is so horribly incomplete, that maybe having it there does more damage than good. If people see that short list they think "oh great" and then there is a million more incompatible changes. Anyway, the previous version `v2.1.0` was python 2 and `aiida-core==0.12.0` only. Doubt that there are people on those versions that would really consider upgrading.

Reply:
Ooh that's right.. most people probably upgrade from some alpha version anyway.